realm/realm-library/build.gradle

```diff
@@ -37,6 +37,7 @@ repositories {
 
 dependencies {
     provided 'io.reactivex:rxjava:1.1.0'
+    provided 'junit:junit:4.12'
```

Comment:
I feel the same as it is essentially only a class needed for debug builds. However the only alternative I can come up with is creating a whole new distribution package for it which also seems like overkill especially since it is so little code in the class.